pyas2/models.py

```diff
@@ -280,6 +284,7 @@ def as2partner(self):
             "enc_alg": self.encryption,
             "mdn_mode": self.mdn_mode,
             "mdn_digest_alg": self.mdn_sign,
+            "canonicalize_as_binary": bool(self.canonicalize_as_binary),
```

Comment:
can you add a test for this

Reply:
You mean add some basic tests to models ? Start with partners and then extend to Organization etc ? This PR is basically not adding any functionality beside providing an attribute to the model/partner which is used in pyas2-lib.

Reply:
lets add an end to end test where canonicalize_as_binary will be used

Reply:
Have added a test. As we are not using canonicalize as binary on sending (I could not find any reference that suggests this should be done and there would be quite some implications on pyas2-lib), I had to mock a couple of things in the test to make it reproduce the issue that I had seen for which this functionality was brought in.